### PR TITLE
Fixed processing of job submitter

### PR DESCRIPTION
**Pull request by blublinsky**
## Why are these changes needed?

Fixes errors introduced by PR that added job submitter to API server

## Checks

- [x ] I've made sure the tests are passing. 
- Testing Strategy
   - [x ] Unit tests
   - [ x] Manual tests
   - [ ] This PR is not tested :(

### CODE REVIEW

**Comment by blublinsky** [comment]
@z103cb, @astefanutti please, take a look

cc @kevin85421

**Review by astefanutti — APPROVED** [review]
LGTM

File: apiserver/pkg/util/job.go [hunk 1]
```diff
@@ -52,9 +48,19 @@ func NewRayJob(apiJob *api.RayJob, computeTemplateMap map[string]*api.ComputeTem
...
+			if err != nil {
+				return nil, errors.New("cpu for ray job submitter is not specified correctly")
+			}
 			cpus = apiJob.JobSubmitter.Cpu
 		}
 		if apiJob.JobSubmitter.Memory != "" {
+			// Ensure that memory size is formatted correctly
+			_, err := resource.ParseQuantity(apiJob.JobSubmitter.Memory)
+			if err != nil {
+				return nil, errors.New("memory for ray job submitter is not specified correctly")
```
Comment (z103cb) [inline apiserver/pkg/util/job.go:62]: why to wrap when creating the new error?
Something like this
```golang
return nil, fmt.Errorf("memory for ray job submitter is not specified correctly: %w", err
```

Comment (blublinsky) [reply apiserver/pkg/util/job.go]: fixed

**Review by z103cb — APPROVED** [review]
LGTM

**Review by kevin85421 — APPROVED** [review]

**Pull request merged**